draft-ietf-ppm-dap.md

```diff
@@ -1263,11 +1262,11 @@ violation. Note that this is also enforced by the Helper during the aggregation
 interaction. The Leader MAY also abort the upload interaction and alert the
 Client with error `reportRejected`.
 
-The Leader MAY ignore any report whose timestamp is before the task's
+The Leader MUST ignore any report whose timestamp is before the task's
 `task_start`, or is past the end time `task_start + task_duration`. When it does
 so, it SHOULD also abort the upload interaction and alert the Client with error
-`reportRejected`. Client MAY choose to opt out of the task if its own clock is
-earlier than `task_start`, or is past `task_start + task_duration`.
+`reportRejected`. Client MUST NOT upload a report if its own clock is earlier
```

Comment:
We say that client's shouldn't upload a report if their own clock is outside of the appropriate interval, but the timestamp in a report doesn't necessarily correspond to the client's clock.

Maybe something like this?

> Clients MUST NOT upload a report if the timestamp included in the report would be earlier than `task_start` or past `task_start + task_duration`.

Reply:
Done.